### DISCUSSION TITLE
Autochop and elven agreements

### DISCUSSION THREAD

**Discussion by Weizen1988** [DFHack/dfhack#3822] [Ideas]
Hello, this is my first time suggesting anything here so hopefully this is the right place. If it isnt could someone please direct me to the right place? Thank you. To get to the point though, I wanted to ask if/suggest that Autochop could  be given a toggle to take into account existing agreements with elves on limiting tree cutting. I've had to disable it most of the time because I keep being given extremely low limits on logging, but its such a useful feature I would like to be able to use it more often.

**Comment by myk002** [comment]
Yes, this is a good idea, and is definitely on our radar as "good thing to implement when we get a chance". There's an issue open for it here: https://github.com/DFHack/dfhack/issues/1269

**Reply by Weizen1988** [reply]
Oh my, Im sorry, I must have missed  that one when I looked. Thank you.
